conf/messages

```diff
@@ -4716,21 +4716,23 @@ representee.cyaChange.isFirstReturn=if this is the first return in this tax year
 
 gainOrLossAfterReliefs.caption=Gain or loss after reliefs
 
-gainOrLossAfterReliefs.title=Tell us if you made a gain on the disposal after reliefs were subtracted
-gainOrLossAfterReliefs.agent.title=Tell us if your client made a gain on the disposal after reliefs were subtracted
-gainOrLossAfterReliefs.trust.title=Tell us if the trust made a gain on the disposal after reliefs were subtracted
-gainOrLossAfterReliefs.capacitor.title=Tell us if the person made a gain on the disposal after reliefs were subtracted
-gainOrLossAfterReliefs.personalRep.title=Tell us if the person made a gain on the disposal after reliefs were subtracted
-gainOrLossAfterReliefs.personalRepInPeriodOfAdmin.title=Tell us if the estate made a gain on the disposal after reliefs were subtracted
-gainOrLossAfterReliefs.personalRepInPeriodOfAdmin.agent.title=Tell us if your client, or the estate made a gain on the disposal after reliefs were subtracted
-
-gainOrLossAfterReliefs.multipleDisposals.title=Tell us if you made a gain on the disposals in this return after reliefs were subtracted
-gainOrLossAfterReliefs.agent.multipleDisposals.title=Tell us if your client made a gain on the disposals in this return after reliefs were subtracted
-gainOrLossAfterReliefs.trust.multipleDisposals.title=Tell us if the trust made a gain on the disposals in this return after reliefs were subtracted
-gainOrLossAfterReliefs.capacitor.multipleDisposals.title=Tell us if the person made a gain on the disposals in this return after reliefs were subtracted
-gainOrLossAfterReliefs.personalRep.multipleDisposals.title=Tell us if the person made a gain on the disposals in this return after reliefs were subtracted
-gainOrLossAfterReliefs.personalRepInPeriodOfAdmin.multipleDisposals.title=Tell us if the estate made a gain on the disposals in this return after reliefs were subtracted
-gainOrLossAfterReliefs.personalRepInPeriodOfAdmin.agent.multipleDisposals.title=Tell us if your client, or the estate made a gain on the disposals in this return after reliefs were subtracted
+gainOrLossAfterReliefs.title=Enter your gain or loss after reliefs
```

Comment:
shouldn't the existing page be shown if no calculation was done?

Reply:
https://jira.tools.tax.service.gov.uk/browse/CPDP-2938 - content changes are part of this.